jumanji/environments/__init__.py

```diff
@@ -42,6 +43,7 @@
 from jumanji.environments.routing.maze.env import Maze
 from jumanji.environments.routing.mmst.env import MMST
 from jumanji.environments.routing.multi_cvrp import MultiCVRP
+from jumanji.environments.routing.pacman.env import PacMan
```

Comment:
What is the reason for naming the environment `pacman` in lowercase and `PacMan` in uppercase? Either the environment is `PacMan` and hence the folder should be `pac_man` or we use `Pacman` and `pacman`. Does this make sense?